client-participation/polis.config.template.js

```diff
@@ -14,6 +14,10 @@ module.exports = {
   //SERVICE_URL: "http://localhost:5000", // local server; recommended for dev
   SERVICE_URL: "http:localhost:5000",
 
+  // Used for setting appropriate hostname for embedding.
+  //SERVICE_HOSTNAME: "123.45.67.89.xip.io",
+  SERVICE_HOSTNAME: "localhost",
```

Comment:
See other review comment.

```suggestion
  // Used for setting appropriate hostname for embedding.
  //SERVICE_HOSTNAME: "123.45.67.89.xip.io",
  SERVICE_HOSTNAME: "localhost",
```